bindings/matrix-sdk-crypto-js/src/identities.rs

```diff
@@ -85,6 +85,28 @@ impl OwnUserIdentity {
 
         future_to_promise(async move { Ok(me.trusts_our_own_device().await?) })
     }
+
+    /// Get the master key of the identity.
+    #[wasm_bindgen(getter, js_name = "masterKey")]
+    pub fn master_key(&self) -> Option<String> {
+        let master_key = self.inner.master_key().as_ref().to_owned();
```

Comment:
No need to taking ownership and then borrow in the next line.
```suggestion
        let master_key = self.inner.master_key().as_ref();
```